lib/core/src/persist/mod.rs

```diff
@@ -108,6 +130,17 @@ impl Persister {
             sync_trigger,
         };
 
+        if let Some(backup_con) = backup_con {
+            if let Err(e) = {
```

Comment:
Doesn't this need to be a closure for this error handling to work on a block level?

Reply:
Oops! Fixed in [0832da8](https://github.com/breez/breez-sdk-liquid/pull/851/commits/0832da8fe3b5189723ef48f16a87d8d2a22f4bc8)